package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
OK so that's this file used for and how is it generated?

Reply:
Google “what is package-lock.json”: package-lock. json is automatically generated for any operations where npm modifies either the node_modules tree, or package. json . It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.

— it ensures that we know the “exact” versions of software running at a point in time.